kusd/validator/states.go

```diff
@@ -71,7 +70,7 @@ func (val *validator) notLoggedInState() stateFn {
 		}
 	} else {
 		// sanity check
-		isVoter, err := val.network.IsVoter(&bind.CallOpts{}, val.walletAccount.Account().Address)
```

Comment:
The `walletAccount` now seems to be stored in the `validationNetwork` right? What if that value changes in the `validator` but the network didn't get the change? in other words, are we making sure the `network` uses the same wallet account as the validator? I think calling functions that update the `walletAccount` like the `SetCoinbase` won't update the `network` object with the new account.

Reply:
you are absolutely right, I just ignored that :)

I can quick fix, but it need bigger changes, next refactor...